README.md

```diff
@@ -18,6 +18,7 @@ Add your plugin to [the `plugins.json` file](https://github.com/serverless/commu
 - Do not remove or modify this section. Make all updates to plugins.json -->
 | Plugin | Stats |
 |:---------------------------|:-----------:|
+| **[@Digitalmaas/serverless Plugin Sqs Alarms - `@digitalmaas/serverless-plugin-sqs-alarms`](https://github.com/digitalmaas/serverless-plugin-sqs-alarms)** <br/> by [digitalmaas](http://github.com/digitalmaas) <br/> Simplifies the setup of CloudWatch Alarms to monitor the visible messages in an SQS queue. | ![Github Stars](https://img.shields.io/github/stars/digitalmaas/serverless-plugin-sqs-alarms.svg?label=Stars&style=for-the-badge) <br/> ![NPM Downloads](https://img.shields.io/npm/dt/@digitalmaas/serverless-plugin-sqs-alarms.svg?label=Downloads&style=for-the-badge)|
```

Comment:
Is there a reason for this being present in your change?

Reply:
Awesome :slightly_smiling_face: I just ran npm run docs on the master branch and it was added again, looks like it was added to plugins.json but not the README in this [commit] maybe?(https://github.com/serverless/plugins/commit/359202bae3226cb60813d82d30ac2af70f7a8127)?

Reply:
Thanks, I believe that's the case here :+1: